### ISSUE TITLE
Inline tags not working

### ISSUE THREAD

**Issue by anthonyhendrickson** [kylekatarnls/jade-php#14]
Jade 1.0+ syntax incorporated inline-tags similar to #{string interpolation}, but with square brackets (i.e., #[span.class Lorem Ipsum] ).

**Comment by kylekatarnls** [comment]
It can be added easily but I don't find this kind of interpolation in the doc. Can you provide a source?

**Comment by anthonyhendrickson** [comment]
Thanks, Kyle. This really comes in handy when you need to wrap only a word, or two, in `<strong>`, `<emph>`, `<a>`, or a special-class `<span class="foo"> ` tag inline with a heading, or paragraph... or at least, this was my experience.

e.g.,
```jade
extends foo

block bar
  h1 J'aime #[strong Jade] beaucoup
  p.
    This is an #[emph example] paragraph. For more information on #[span.foo whatever],
    click #[a(href="#") here]!
```

**Comment by kylekatarnls** [comment]
Qualified: It will be available in jade-php 1.7

**Comment by kylekatarnls** [comment]
Hi, 1.7.0-rc1 is out and implement inline tags. Do not hesitate to test it and report any problem or suggestion.

Known bug: ```click #[a(href="#") here]!``` will create a space before the exclamation point with prettyprint = true, this bug previously exist and need some work to get the same result as in jade-js (that is: all inline tags have any spaces inside or outside).

**Comment by kylekatarnls** [comment]
The version 1.7.0 is released with 38424733b92e24c020fe93b71ace9169cde15c49

The spaces problem will be fixed with #56 on the 1.8.0
